src/gps/gps.c

```diff
@@ -177,6 +201,7 @@ void GPS_sample_update(GpsSample *newSample)
     updateFullDateTime(newSample);
     g_gpsSnapshot.deltaFirstFix = newSample->time - g_timeFirstFix;
     g_gpsSnapshot.previousPoint = prevPoint;
+    g_gpsSnapshot.bearing = gps_bearing(&prevPoint, &(newSample->point));
```

Comment:
ah, I think this is what's causing the spikey behavior, calculating it across adjacent GPS points.

Consider running it through our Exponential Moving Average IIR filter - see filter.c. it's what we use to apply smoothing to our analog and timer/pulse channels.